Sasquatch/Sasquatch/Info.plist

```diff
@@ -16,14 +16,12 @@
 	<string>APPL</string>
```

Comment:
Looks like this info.plist under Sasquatch/ is not used and can be deleted. There is already two other under each apps